MockKit/MockCGMManager.swift

```diff
@@ -112,26 +112,53 @@ public final class MockCGMManager: TestingCGMManager {
 
     public let shouldSyncToRemoteService = false
 
+    private func logDeviceComms(_ type: DeviceLogEntryType, message: String) {
+        delegate.notify { (delegate) in
```

Comment:
[nit] Consistent use of both `delegate` vs. `self.delegate` and `{ (delegate) in` vs. `{ delegate in` for this line and the similar line in `sendCGMResult` below.